src/support/timestamp.c

```diff
@@ -406,7 +399,7 @@ __time_value_validate_parent(
           __wt_time_window_to_string(tw, time_string[0]),
           __wt_time_aggregate_to_string(parent, time_string[1]));
 
-    if (tw->start_ts < parent->oldest_start_ts)
+    if (tw->start_ts != WT_TS_NONE && tw->start_ts < parent->oldest_start_ts)
```

Comment:
Same comment as before, false positive every time `start_ts` is `0`.

Reply:
The new check is added to prevent the scenario where the child page does not have any timestamps, in that case, the child `oldest_start_ts` is 0 whereas because of another child, the parent can have a valid `oldest_start_ts`. Due to that reason, we added a check that the `oldest_start_ts` cannot be 0.